app/src/main/res/layout/recently_played_story_card.xml

```diff
@@ -65,16 +65,16 @@
           android:ellipsize="end"
           android:lines="2"
           android:text="@{viewModel.storyTitle}"
-          android:textColor="@color/component_color_recently_played_activity_recently_played_story_card_text_color"
+          android:textColor="@color/component_color_shared_content_item_text_view_color"
           android:textStyle="bold"
           app:layout_constraintEnd_toEndOf="parent"
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
           android:id="@+id/topic_name_text_view"
-          style="@style/CaptionViewStart"
```

Comment:
This style is keeping the text in bold stage but we not need bold text Story Name as per mock.

Reply:
Keep
`style="@style/CaptionViewStart"`
as well as 
`android:fontFamily="sans-serif-light"`

Reply:
Okay